### PR TITLE
add the third database

### PR DESCRIPTION
**Pull request by nickemma**
In this milestone, I achieved the following...

- I create tables named vets, visits, and specialization.
- I input data into vet, visits, and specialization accordingly.
- I wrote queries to check the data according to the requirement.
- The snapshots are below...

join1
join2
join3
join4
join5
join6

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,4 +6,42 @@ CREATE TABLE animals (
...
+);
+
+ALTER TABLE animals ADD species varchar(100);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD species_id BIGINt REFERENCES species(id);
+
+ALTER TABLE animals ADD owners_id BIGINT REFERENCES owners(id);
+
```
Comment (CollinsTatang) [inline schema.sql]: ### **Optional**
- [x] Please remove all unnecessary code from the `schema.sql` file

File: queries.sql [hunk 2]
```diff
@@ -35,4 +35,61 @@ SELECT COUNT(*) FROM animals WHERE escape_attempt = 0;
...
+UPDATE animals SET owners_id = 1 WHERE name = 'Agumon';
+
+UPDATE animals SET owners_id = 2 WHERE name  = 'Gabumon' OR name ='Pikachu';
+
+UPDATE animals SET owners_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+
+UPDATE animals SET owners_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+
+UPDATE animals SET owners_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
```
Comment (CollinsTatang) [inline queries.sql]: ### **Optional**
- [x] Please remove all unnecessary code from the `queries.sql` file

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (nickemma) [reply schema.sql]: okay

File: queries.sql (hunk 2 above)
Comment (nickemma) [reply queries.sql]: okay senior

**Pull request merged**